### ISSUE TITLE
Carryover `line-height`s are affected element alignment

### ISSUE THREAD

**Issue by mattnolting** [patternfly/patternfly#6876]
# TLDR
There are several `line-height: 1;` definitions in our codebase, and after updating to `flex/inline-flex` `grid/inline-grid` `line-height: 1;` affects element positioning. 

After a quick search, here are several instances that should be reviewed:

- [src/patternfly/components/Icon/icon.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Icon/icon.scss#L52)
- [src/patternfly/components/Breadcrumb/breadcrumb.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Breadcrumb/breadcrumb.scss#L65)
- [src/patternfly/components/Form/form.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Form/form.scss#L321)
- [src/patternfly/components/Slider/slider.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Slider/slider.scss#L135)
- [src/patternfly/components/MenuToggle/menu-toggle.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/MenuToggle/menu-toggle.scss#L588)
- [src/patternfly/components/Table/table-grid.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Table/table-grid.scss#L498)
- [src/patternfly/components/Nav/nav.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Nav/nav.scss#L1056)
- [src/patternfly/components/Tabs/tabs.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Tabs/tabs.scss#L741)
- [src/patternfly/components/CalendarMonth/calendar-month.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/CalendarMonth/calendar-month.scss#L190)
- [src/patternfly/components/Chip/chip.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/Chip/chip.scss#L131)
- [src/patternfly/components/FormControl/form-control.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/FormControl/form-control.scss#L126)
- [src/patternfly/components/List/list.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/List/list.scss#L55)
- [src/patternfly/components/EmptyState/empty-state.scss](https://github.com/patternfly/patternfly/blob/50182be1284c32d664c28b32bb348d50ce334c26/src/patternfly/components/EmptyState/empty-state.scss#L105)
